src/benchcab/internal.py

```diff
@@ -274,3 +274,12 @@ def get_met_forcing_file_names(experiment: str) -> list[str]:
     ]
 
     return file_names
+
+# Configuration for the client upload
+MEORG_CLIENT = dict(
```

Comment:
New internals keys for client, note the `copyq` only allows a single thread, boo.